src/app/ipv/middleware.js

```diff
@@ -83,6 +83,7 @@ module.exports = {
         /^\/journey\/(next|error|fail)$/,
         /^\/journey\/cri\/build-oauth-request\/(ukPassport|stubUkPassport|fraud|stubFraud|address|stubAddress|kbv|stubKbv|activityHistory|stubActivityHistory|dcmaw|stubDcmaw|debugAddress)$/,
         /^\/journey\/build-client-oauth-response$/,
+        /^\/journey\/retrieve-cri-credential$/,
```

Comment:
I think this needs changing to match the api-gateway endpoint right? Is it /journey/cri/credential

Reply:
Correct! Will close the PR!